basics/a77.go

```diff
@@ -0,0 +1,26 @@
+/* Constants and Type Declarations
+*/
+package main
+
+import (
+	"fmt"
+	"time"
+)
+
+const fiveSeconds = 5 * time.Second
+
+func main() {
+	now := time.Now()
+	FiveNanoSeconds1 := now.Add(-5) 
+	FiveNanoSeconds2 := now.Add(-fiveSeconds)
+	// if we declare integer -5
+	myvar := -5
+	FiveNanoSeconds3 := now.Add(myvar) // will give compiler erro
```

Comment:
./a77.go:18:29: cannot use myvar (type int) as type time.Duration in argument to now.Add

Reply:
FiveNanoSeconds3 is unused